apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManagerIT.java

```diff
@@ -77,7 +77,6 @@ public void emptyFilters() {
               assertFalse(filterManager.isResultSetEmpty(INVALID_BENE, testBound));
 
               // Refresh the filter list
-              filterManager.refreshFilters();
```

Comment:
why is this removed?

Reply:
yea, it shouldn't have been; thx for catching that!